### ISSUE TITLE
Utilizing this for multiclass custom datasets

### ISSUE THREAD

**Issue by SujithSash** [mathilde-b/SFDA#7]
Can this be utilized straightway for muticlass source and target datasets. If not where exactly are we supposed to make changes with respect to the loss functions?Thanks in advance!

**Comment by mathilde-b** [comment]
Hello Sujith, yes of course! it works for multiclass. --n_class is the class number parameter.
Your classes should be labeled 0 (background), 1, 2, 3, ..., 
Let me know!
Mathilde

**Comment by SujithSash** [comment]
Thanks for the response Mathilde. Yeah, I believe I shouldn't have a problem with the weighted shanon entropy loss(Lent). But coming to the KL divergence loss which needs knowledge of anotomical priors which I don't have in my custom use case unfortunately, can I just use the standard cross entropy loss instead based on the generated psuedolabels on my target images?Basically, shanon loss plus the Cross entropy loss on the target domain predictions as my ground truth during the adaptation phase(retraining) is what I mean. Will be glad to hear your opinion. Thanks in advance!

**Comment by mathilde-b** [comment]
Hello again, what is your application ? sometimes even a very coarse prior works better than no prior on the size of the target regions you wish to segment. 
Otherwise, yes I think this could work, I have never tried it though. Would be interested to know the results !
